### ISSUE TITLE
[Bug]:  WEB:  Unsupported operation: Platform._operatingSystem

### ISSUE THREAD

**Issue by sgehrman** [hm21/pro_image_editor#35]
### Package Version

2.7.2

### Flutter Version

3.19

### Platforms

Web

### How to reproduce?

make an emoji and drag it around.  Drag is also kind of broken.

### Logs (optional)

```sh
══╡ EXCEPTION CAUGHT BY GESTURE ╞═══════════════════════════════════════════════════════════════════
The following UnsupportedError was thrown while handling a gesture:
Unsupported operation: Platform._operatingSystem
```

### Example code (optional)

mouse is not visible, but it's on the right side
![bug](https://github.com/hm21/pro_image_editor/assets/287137/271ea71c-0e3e-41a5-984d-cadec1f95031)

Also, adding text makes a full screen barrier, but the image isn't maximized like drawing.
![Screenshot-20240504193010-2917x1794](https://github.com/hm21/pro_image_editor/assets/287137/ea8117a8-8a95-4063-80e8-d5eab1a997c7)

### Device Model (optional)

linux

**Comment by sgehrman** [comment]
I assume you need to use: universal_io: ^2.2.2

**Comment by hm21** [comment]
Thanks again, Steve for reporting an issue.  

I found the problem with the platform error. I had a part in the code that looks like this `if (Platform.isAndroid)` and you right `universal_io` will resolve this issue and make it a bit easier for me to program the package. But I try to use not many other packages, so I just changed the code to this `if(!kIsWeb && Platform.isAndroid)` which should also resolve this issue.

To the part with “mouse is not visible, but it's on the right side”:
Yes, in the current version I had the problem that layers do not detect correct gestures when they are overlaid on the background image. However, in the `dev branch` I have already completely rewritten the way they are detected, which should solve this problem. I can't say when I will merge the dev branch into the stable branch because I still have to write more code for the new crop-rotate editor, but I hope to release it in the next weeks.
